### ISSUE TITLE
(->snake_case :s3-key) ; => :s_3_key

### ISSUE THREAD

**Issue by rsslldnphy** [qerub/camel-snake-kebab#22]
Is there any way to avoid numbers creating separators? Had a quick scan of the code but couldn't see an obvious workaround. Thanks.

**Comment by qerub** [comment]
I've created pull request #23 implementing the previously mentioned approach. Feedback is more than welcome.

**Comment by qerub** [comment]
The just released version 0.3.0 allows you to do `(->snake_case :s3-key :separator \-)`.

I'm leaving this issue open because I want a better solution.

**Comment by kenrestivo-stem** [comment]
There's a similar annoying problem with kebab-case"

```clojure
  (csk/->kebab-case :s3_bucket_name)
;; >  :s-3-bucket-name
```

**Comment by agzam** [comment]
Is there any update on this issue? I mean it's been hanging here since 2014 😱

**Comment by vincent-dm** [comment]
Yeah, I'd also very much like a way to disable letters followed by numbers being separated.

**Comment by waffletower** [comment]
A reminder that snake -> kebabs transforms containing non-leading numbers are not reversible:

```clojure
(-> "a1_example"
          ->kebab-case-keyword
          ->snake_case_string)
"a_1_example"
user> (-> "a2a_example"
          ->kebab-case-keyword
          ->snake_case_string)
"a_2a_example"
